src/almanac/verdant-sundial/verdant-sundial.md

```diff
@@ -0,0 +1,40 @@
+# Verdant Sundial
+
+```{figure} ../assets/Sundial_2916_HDR.jpeg
+---
+alt: Sundial on a gravestone.
+---
+_The verdant sundial casts shadows of past and future intention observed in the present. (Image source: {cite:alp}`image_elfward_sundial`)._
+```
+
+> "The present is the ever moving shadow that divides yesterday from tomorrow. In that lies hope." <br>
+> \- Frank Lloyd Wright
+
+The software garden is intertwined in a story of time which is exhibited like a shadow on a moss-covered sundial.
+Time is observed indirectly through a kind of software archeology in past-tense material artifacts (files) and present-tense operation (procedures).
+These artifacts act as shadows of our prior creative intentions where procedures become the living present materialized promise of those intentions.
+"This brings an uncanny sense of living in a state where the future is a dimension of our present reality." {cite:p}`morton2013hyperobjects`
+The future is elusive but present in software artifacts, similar to rings of a tree that depict the hope of change.
+
+These software tree rings are vessels of temporal data which offer a unique insight into growth patterns.
+We can study these patterns like dendronchronology: "Peel away the hard, rough bark and there is a living document, history recorded in rings of wood cells." {cite:p}`gramling2020tree`
+This helps us understand the chronological nature of software and how it evolves.
+It also shares patterns of temporal pulsing - not all times of the software garden are the same.
+
+These software pulses give us with visibility into distinct growing or receding patterns.
+Some code exhibits annual behavior, requiring continual replanting efforts each season.
+Others are perennials, undergoing seasonal change perpetuated by periods of vibrancy and dormancy.
+Seasons of software promote differing patterns among these; some systems prepare for winter through conservation only to fervently bloom during their spring.
+
+Seasonal chronology in software benefits also from an awareness of kairos, the opportune timing of our actions amidst time.
+Each software gardening moment includes a series of options of varying kairological fitness.
```

Comment:
I think an elaboration sentence before this might also help clarify. What is a software gardening moment? Maybe tie together awareness of seasonal chronology as a software gardening skill?

Reply:
Ah!! You do this in the very next sentence. Maybe my suggestion is distracting then, but you could also consider rearranging this paragraph